File: problem1-5/problem1/problem1.py
#!/usr/bin/env python
# Add all the natural numbers below one
# thousand that are multiples of 3 or 5.
def soln():
    ans = sum([i for i in xrange(1,1000) if (i % 5 == 0) or (i % 3 == 0)])
    print("Solution: {0}".format(ans))


if __name__ == "__main__":
    soln()

File: problem1-5/problem1/problem1.go
package main

func main() {
    ans := 0
    for i := 0; i < 1000; i++ {
        if i % 3 == 0 {
            ans += i
        } else if i % 5 == 0 {
            ans += i
        }
    }
    println("Solution:", ans)
}

